modules/browser/BrowserCapabilities.js

```diff
@@ -0,0 +1,288 @@
+import bowser from 'bowser';
+
+import BrowserDetection from './BrowserDetection';
+import capabilitiesDB from './capabilities.json';
+
+// TODO: All methods should use capabilities.json when possible.
+
+// NOTE: Now we are extending BrowserDetection in order to preserve
+// RTCBrowserType interface but maybe it worth exporting BrowserCapabilities
+// and BrowserDetection as separate objects in future.
+
+/**
+ * Implements browser capabilities for lib-jitsi-meet.
+ */
+export default class BrowserCapabilities extends BrowserDetection {
+    /**
+     * Creates new BrowserCapabilities instance.
+     *
+     * @param {boolean} [isUsingIFrame] - True if Jitsi Meet is loaded in iframe
+     * and false otherwise.
+     * @param {Object} [browserInfo] - Information about the browser.
+     * @param {string} browserInfo.name - The name of the browser.
+     * @param {string} browserInfo.version - The version of the browser.
+     */
+    constructor(isUsingIFrame = false, browserInfo) {
+        super(browserInfo);
+
+        const name = this.getBrowserName();
+        const version = this.getBrowserVersion();
+
+        const browserCapabilities = capabilitiesDB[name] || [];
+
+        const capabilitiesByVersion = browserCapabilities.find(entry =>
+            !entry.version || bowser.compareVersions([
```

Comment:
Can we use on of the wrappers here (e.g. this.isCurrentVersionGreaterThan)?

Reply:
Yes. Awesome idea! Thanks! I tried it and it simplifies a lot the code (now bowser import and constants for name and version are gone)!